packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -14,3 +14,16 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-one-time-charge = One-time charge (due today)
+
+# Upgrade is a noun, as in "cost for upgrade", and is followed by the price
+sub-update-prorated-total = <strong>Upgrade</strong> (Prorated to end of period)
```

Comment:
packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx#L161-162 shows this using a `<span className="font-semibold">Upgrade</span>` instead of a `<strong>Upgrade</strong>`:
```html
<span className="font-semibold">Upgrade</span> (Prorated to end of period)
```